SmartDeviceLink/SDLManagerDelegate.h

```diff
@@ -55,7 +55,19 @@ NS_ASSUME_NONNULL_BEGIN
  * @param language The language of the connected head unit the manager is trying to update the configuration.
  * @return An object of SDLLifecycleConfigurationUpdate if the head unit language is supported, otherwise nil to indicate that the language is not supported.
  */
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
 - (nullable SDLLifecycleConfigurationUpdate *)managerShouldUpdateLifecycleToLanguage:(SDLLanguage)language;
+#pragma clang diagnostic pop
```

Comment:
This is incorrect. This changes it so that if developers are implementing the old method, they don't get a deprecation warning, but it should show a deprecation warning to the developer. This needs to move to the point that the comment above was so that a warning is not thrown in the SDL project when devs integrate it.